perma_web/perma/templates/docs/accounts.html

```diff
@@ -0,0 +1,76 @@
+{% extends "base-responsive.html" %}
+{% block title %} | Docs{% endblock %}
+
+{% block meta_description %}
+This section of the user guide describes how Perma accounts work.
+{% endblock %}
+
+{% block mainContent %}
+
+<div class="container cont-fixed">
+  <div class="row">
+    <div class="col-sm-12">
+      <h1>Perma.cc user guide <span class="subsection">Accounts</span></h1>
+    </div>
+  </div>
+</div>
+
+{% include 'docs/docs_menu.html' %}
+
+<div class="container cont-reading cont-fixed" id="overview">
+  <div class="row">
+    <div class="col-sm-4 reading-toc" >
+      <ul>
+        <li class="reading-toc-chapter"><a href="#orgs">Accounts</a></li>
+        <li class="reading-toc-section"><a href="#who-needs-account">Who Needs an Account?</a></li>
+        <li class="reading-toc-section"><a href="#">Usage Limits</a></li>
+        <li class="reading-toc-chapter"><a href="#orgs">Organizations</a></li>
+        <li class="reading-toc-section"><a href="#free-service-academic">Free Service for Academic Organizations</a></li>
+        <li class="reading-toc-section"><a href="#free-service-courts">Free Service for Courts</a></li>
+        <li class="reading-toc-section"><a href="#paid-service">Paid Service for Other Organizations</a></li>
+        <li class="reading-toc-section"><a href="#authority">Authority and Responsibilities</a></li>
+        <li class="reading-toc-section"><a href="#adding-org-users">Adding and Removing Organization Users</a></li>
+      </ul>
+    </div>
+
+    <div class="col-sm-8 reading-body">
+      
+      <h2 class="body-ah">Accounts</h2>
```

Comment:
In the TOC, you link to this section using an href. So, that means this heading needs an id for the href to target, for instance "accounts". `<h2 id="accounts" class="body-ah">Accounts</h2>`